tee-worker/ts-tests/identity.test.ts

```diff
@@ -72,28 +131,133 @@ describeLitentry("Test Identity", (context) => {
             console.log("post verification msg to twitter: ", msg);
             assert.isNotEmpty(challengeCode, "challengeCode empty");
         }
+        //create ethereum identity
+        const resp_ethereum = await createIdentity(
+            context,
+            context.defaultSigner,
+            aesKey,
+            true,
+            ethereumIdentity
+        );
+        if (resp_ethereum) {
+            const [_who, challengeCode] = resp_ethereum;
+            console.log("ethereumIdentity challengeCode: ", challengeCode);
+            const msg = generateVerificationMessage(
+                context,
+                hexToU8a(challengeCode),
+                context.defaultSigner.addressRaw,
+                ethereumIdentity
+            );
+            console.log("post verification msg to ethereum: ", msg);
+            ethereumValidationData!.Web3Validation!.Evm!.message = msg;
+            const msgHash = ethers.utils.arrayify(msg);
+            signature_ethereum = await context.ethersWallet.alice.signMessage(msgHash);
+            ethereumValidationData!.Web3Validation!.Evm!.signature!.Ethereum = signature_ethereum;
+            assert.isNotEmpty(challengeCode, "challengeCode empty");
+        }
+        // create substrate identity
+        const resp_substrate = await createIdentity(
+            context,
+            context.defaultSigner,
+            aesKey,
+            true,
+            substrateIdentity
+        );
+        if (resp_substrate) {
+            const [_who, challengeCode] = resp_substrate;
+            console.log("substrateIdentity challengeCode: ", challengeCode);
+            const msg = generateVerificationMessage(
+                context,
+                hexToU8a(challengeCode),
+                context.defaultSigner.addressRaw,
+                substrateIdentity
+            );
+
+            console.log("post verification msg to substrate: ", msg);
+            substrateValidationData!.Web3Validation!.Substrate!.message = msg;
+            signature_substrate = context.defaultSigner.sign(msg);
+            substrateValidationData!.Web3Validation!.Substrate!.signature!.Sr25519 =
```

Comment:
qq: when would this be ED255 or EDCA?

Reply:
@jonalvarezz I don't understand your question, can you explain it more clearly?:see_no_evil: